moe/cli.py

```diff
@@ -24,13 +24,19 @@
 
 console = Console()
 
+# define logging levels based on verbose/quiet flags
+VERBOSE_INFO_LEVEL = 1
+VERBOSE_DEBUG_LEVEL = 2
+QUIET_ERROR_LEVEL = 1
+QUIET_CRITICAL_LEVEL = 2
```

Comment:
Just wondering how this works. Is the idea to build into the logger that `--verbose` means both `INFO` and `DEBUG` but quiet (`--quiet`?) means error and critical only?

Reply:
So default is warning. `-v` is error. `-vv` is debug. `-q` is error and `-qq` is critical.